coresimd/src/x86/i586/avx2.rs

```diff
@@ -1257,7 +1257,7 @@ pub unsafe fn _mm256_shuffle_epi8(a: u8x32, b: u8x32) -> u8x32 {
 /// # #![feature(cfg_target_feature)]
 /// # #![feature(target_feature)]
 /// #
-/// # #[macro_use] extern crate stdsimd;
+/// # #[macro_use] extern crate coresimd as stdsimd;
```

Comment:
I was a bit lazy here because I don't know the best way to fix the examples so that they show `coresimd` in the docs of `coresimd` and `stdsimd` in the docs of `stdsimd`. So here I just changed as little as possible for now.